src/mcp/index.ts

```diff
@@ -174,7 +174,8 @@ export class FirebaseMcpServer {
 
   async getAuthenticatedUser(): Promise<string | null> {
     try {
-      return await requireAuth(await this.resolveOptions());
+      const email = await requireAuth(await this.resolveOptions());
+      return email ?? "Application default credentials";
```

Comment:
nit: "Application Default Credentials"

Could you also update the tsdoc on `requireAuth`? It wasn't obvious what `null` means reading it.

Thanks!